app/views/JitsiMeetView/index.tsx

```diff
@@ -17,8 +20,31 @@ const JitsiMeetView = (): React.ReactElement => {
 		params: { rid, url, videoConf }
 	} = useRoute<RouteProp<ChatsStackParamList, 'JitsiMeetView'>>();
 	const { goBack } = useNavigation();
+	const user = useAppSelector(state => getUserSelector(state));
+	const serverUrl = useAppSelector(state => state.server.server);
 
 	const [authModal, setAuthModal] = useState(false);
+	const [cookiesSet, setCookiesSet] = useState(false);
+
+	const setCookies = async () => {
+		const date = new Date();
+		date.setDate(date.getDate() + 1);
```

Comment:
Why only 1 ms? Will the expire time affect the cookie if the device is slow?

Reply:
1 DAY
[setDate](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setDate)